src/components/ImageUpload/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { uploadFileToBlob } from "&server/utils/ImageUpload";
+
+import classes from "./ImageUpload.module.scss";
+
+interface Props {
+  setImageUrl: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ImageUpload: React.FC<Props> = ({ setImageUrl }) => {
+  const [fileSelected, setFileSelected] = useState(null);
+  const [uploading, setUploading] = useState(false);
+
+  const onFileChange = (event: any) => {
+    setFileSelected(event.target.files[0]);
```

Comment:
This is the part I reached out about. We want to pass the url of the previously uploaded image when a user has saved a form for later, but this `setFileSelected` function is what controls the preview and it needs an actual file.